pose-detection/src/calculators/velocity_filter_utils.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+import {Keypoint} from '../types';
+
+// Estimate object scale to use its inverse value as velocity scale for
+// RelativeVelocityFilter. If value will be too small (less than config's
+// `minAllowedObjectScale`) smoothing will be disabled and landmarks will be
+// returned as is.
+// Object scale is calculated as average between bounding box width and height
+// with sides parallel to axis.
+export function getObjectScale(landmarks: Keypoint[]): number {
+  const x = landmarks.map(landmark => landmark.x);
```

Comment:
In general, I expect that just square around for keypoints is not the best way to estimate object scale. And this approach can lead to fluctuations depending on scale. Torso as source of scale or distance between hidden center and rotation keypoints would be more stable way to estimate human pose scale.